.github/workflows/test-publish.yml

```diff
@@ -142,3 +140,5 @@ jobs:
               generate_release_notes: true
             });
             console.log(`Release creation result: ${relRes.status}`);
```

Comment:
**[learning]** When do we use `echo` and when do we use `console.log` in `.yml` files? Are they outputted to different places?

Reply:
Typically when you see commands in a step of a GitHub Action workflow it's bash and so it uses `echo`.  This Action (`actions/github-script@v5`) is special, it takes a `script` property which is a JavaScript script that has access to the GitHub SDK.  So because it's JavaScript it uses `console.log`.